content/providers/quick-start/data.yaml

```diff
@@ -31,26 +31,32 @@
       from qiskit.primitives import Estimator
       estimator = Estimator()
 
-      # Express hydrogen molecule Hamiltonian as an operator
+      # Specify problem hamiltonian
       from qiskit.quantum_info import SparsePauliOp
-      H2_operator = SparsePauliOp.from_list([
-          ("II", -1.052373245772859),
-          ("IZ", 0.39793742484318045),
-          ("ZI", -0.39793742484318045),
-          ("ZZ", -0.01128010425623538),
-          ("XX", 0.18093119978423156)
+      
```

Comment:
```suggestion
```

Reply:
done in 1ea4fcbc676223dd75bb1c3823d74294184792d0